Makefile

```
@@ -7,30 +9,13 @@ type-qualifiers.txt:
 	| sed 's/\(.*\)\.java/    "\1",/' \
 	| LC_COLLATE=C sort | uniq > $@
 
+.PHONY: tags
 TAGS: tags
```

checkmake (0.2.2)

[warning] Target "TAGS" should be declared PHONY.

(phonydeclared)